replaceable_schema.sql

```diff
@@ -0,0 +1,730 @@
+-- This file sets up the `r` schema, which contains things that can be safely dropped and replaced instead of being
```

Comment:
Move this into migrations folder, no need to have it at the top level. Also I think a name like `triggers.sql` would be clearer.

Reply:
I added a new db_schema/src/schema_setup folder that contains this SQL code (now in triggers.sql and utils.sql) and the Rust code that runs all SQL code including migrations.

The migrations folder should only contain migrations.